go/v1beta1/storage/esutil/client.go

```diff
@@ -497,20 +497,20 @@ func (c *client) Update(ctx context.Context, request *UpdateRequest) error {
 		c.esClient.Index.WithRefresh(request.Refresh),
 	)
 	if err != nil {
-		return err
+		return nil, err
 	}
 	if res.IsError() {
-		return fmt.Errorf("unexpected response from elasticsearch: %s", res.String())
+		return nil, fmt.Errorf("unexpected response from elasticsearch: %s", res.String())
 	}
 
 	esResponse := EsIndexDocResponse{}
 	if err := DecodeResponse(res.Body, &esResponse); err != nil {
-		return err
+		return nil, err
 	}
 
 	log.Debug("elasticsearch response", zap.Any("response", esResponse))
 
-	return nil
+	return &esResponse, nil
```

Comment:
Technically the only field we need at the moment is the version, but we may want to grab the sequence number or one of the other fields later.